www/README.md

```diff
@@ -1,16 +1,51 @@
-# React documentation site
-
-To get setup:
-```
-yarn install
-```
-
-To run the dev server:
-```
-yarn run dev
-```
-
-To clear the local cache:
-```
-yarn run reset
-```
\ No newline at end of file
+# React — documentation site
+
+## Getting started
+
+### Prerequisites
+
+1. Git
+1. Node: install version 7.6 or greater
+1. Yarn: `npm i -g yarn` to install it globally via NPM
+1. A clone of the [https://github.com/facebook/react](React repo) on your local machine
```

Comment:
Wrong order in Markdown link (doesn't render)

Reply:
Doh, fixed :)